civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -95,13 +95,11 @@
 // Custom CiviHR Theme CSS
 // --------------------------------------------------
 @import "base/custom";
-
+@import "base/_temp_custom";
```

Comment:
also can you give it a more meaningful name? Like `_task-assignments-modal-temp.scss`